mud-contracts/world/src/modules/kill-mail/systems/KillMail.sol

```diff
@@ -16,7 +16,7 @@ contract KillMail is AccessModified, EveSystem {
   ) public onlyAdmin hookable(killMailId, _systemId()) {
     ResourceId tableId = Utils.killMailTableId(_namespace());
 
-    if (KillMailTable.getKiller(tableId, killMailId) != address(0)) {
+    if (KillMailTable.getKillerCharacterId(tableId, killMailId) != 0) {
```

Comment:
@0xxlegolas this is the right way to check if a kill is already registered, right?

Reply:
Yes this should work